astropy/coordinates/baseframe.py

```diff
@@ -30,15 +30,15 @@
 from .transformations import TransformGraph
 from . import representation as r
 
-from .frame_attributes import FrameAttribute
+from .attributes import Attribute
 
-# Import all other FrameAttributes so we don't break backwards-compatibility
+# Import all other Attributes so we don't break backwards-compatibility
 # (some users rely on them being here, although that is not
 # encouraged, as this is not the public API location.)
-from .frame_attributes import (
-    TimeFrameAttribute, QuantityFrameAttribute,
+from .attributes import (
```

Comment:
Here, for backwards compatibility, below the import you should add:
```python
TimeFrameAttribute = TimeAttribute
QuantityFrameAttribute = QuantityAttribute
CartesianRepresentationFrameAttribute = CartesianRepresentationAttribute
```
so the old names still resolve